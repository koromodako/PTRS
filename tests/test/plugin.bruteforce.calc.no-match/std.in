calc
{
  "bin":"bruteforce",
  "fragment_id":"1",
  "params":{
    "charset":"abcdefghijklmnopqrstuvwxyz0123456789",
    "hash_func":"md5",
    "max_len":1,
    "min_len":1,
    "target":"1bc29b36f623ba82aaf6724fd3b16718"
  }
}
EOF
