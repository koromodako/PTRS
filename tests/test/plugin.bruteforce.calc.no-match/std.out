{
    "fragment_id": "1",
    "result": {
        "has_match": false,
        "match_str": ""
    }
}
